whittle/prunning/pruners/sparsegpt.py

```diff
@@ -0,0 +1,121 @@
+from argparse import Namespace
+import logging
+
+import torch
+from torch.utils.data import DataLoader
+
+from whittle.models.gpt import GPT
+from whittle.modules.linear import Linear
+from whittle.prunning.pruners.base_pruner import Pruner
+from whittle.prunning.utilis.sparsegpt import SparseGPT
+from whittle.prunning.utilis.catcher import Catcher
+
+
+class SparseGptPruner(Pruner):
+    def _prune_wanda_sparse(
```

Comment:
Could you please fix the naming and use something more generic? Using "wanda" for sparsegpt pruner is counterintuitive.